docs/platform/documents/broadcast.md

```diff
@@ -8,4 +8,4 @@ Parameters:
 | **document**              | Document  | yes            | A valid [created document](/platform/documents/create.md) |
 | **identity**              | Identity  | yes            | A valid [registered identity](/platform/identities/register.md) |
 
-Returns : void.
+Returns : document.
```

Comment:
What if i pass many docs?

Reply:
Good question. Especially if there is some that succeed and some that fails. 
Should we returns documents{} input ? or we keep to void ?